pybricks/pupdevices/pb_type_pupdevices_remote.c

```diff
@@ -89,12 +89,12 @@ STATIC void remote_connect(const char *name, mp_int_t timeout) {
     // we are using static memory
     memset(remote, 0, sizeof(*remote));
 
+    remote->context.hub_kind = LWP3_HUB_KIND_HANDSET;
```

Comment:
Not a big deal, but it looks like this line got moved in the second commit for some reason.